dockerfiles/splash/install-pyqt5.sh

```diff
@@ -1,15 +1,14 @@
 #!/usr/bin/env bash
 
 _PYTHON=python3
-SPLASH_BUILD_PARALLEL_JOBS=8
 
 mkdir -p /tmp/builds/sip && \
 mkdir -p /tmp/builds/pyqt5 && \
 pushd /tmp/builds && \
 # sip
 tar xzf "$1" --keep-newer-files -C sip --strip-components 1 && \
 pushd sip && \
-${_PYTHON} configure.py --sip-module PyQt5.sip && \
+${_PYTHON} configure.py --sip-module PyQt${QT_MAJOR_VERSION}.sip && \
```

Comment:
As the file is called install-pyqt5, I'm not sure module name should be generated. We don't know how if PyQt5 is going to change its name or not, when Qt6 is landed.

```suggestion
${_PYTHON} configure.py --sip-module PyQt5.sip && \
```